website/home/app/showcase/showcase-client.tsx

```diff
@@ -0,0 +1,389 @@
+    github:
+      "https://github.com/gensx-inc/gensx/tree/main/examples/openai-computer-use",
+    tags: ["Provider: OpenAI", "Feature: Vision", "Category: Automation"],
+  },
+  {
+    id: "all-models-metadata",
+    title: "Model Metadata Explorer",
+    description:
+      "Workflow that fetches and displays technical specifications, context limits, and pricing for AI models.",
+    github:
+      "https://github.com/gensx-inc/gensx/tree/main/examples/all-models-metadata",
+    tags: ["Category: Reference Tool", "Type: Workflow", "Feature: Model Info"],
+  },
+  {
+    id: "salty-ocean-model-comparison",
+    title: "Model Comparison Tool",
+    description:
+      "Side-by-side comparison interface for testing prompts across multiple AI models simultaneously.",
+    github:
+      "https://github.com/gensx-inc/gensx/tree/main/examples/salty-ocean-model-comparison",
+    tags: ["Category: Developer Tool", "Feature: Multi-model", "Type: Utility"],
+  },
+  {
+    id: "commonjs-compatibility",
+    title: "CommonJS Compatibility",
+    description:
+      "Demonstrates using GenSX with require() syntax in CommonJS environments and Node.js < 14.",
+    github:
+      "https://github.com/gensx-inc/gensx/tree/main/examples/commonjs-compatibility",
+    tags: ["Environment: CommonJS", "Platform: Node.js", "Type: Compatibility"],
+  },
+  {
+    id: "typescript-compatibility",
+    title: "TypeScript Examples",
+    description:
+      "TypeScript configuration and usage patterns for GenSX with strict type checking enabled.",
+    github:
+      "https://github.com/gensx-inc/gensx/tree/main/examples/typescript-compatibility",
+    tags: ["Language: TypeScript", "Feature: Type Safety", "Type: Examples"],
+  },
+  {
+    id: "nextjs-ai-chatbot-template",
+    title: "Next.js AI Chatbot Template",
+    description:
+      "Production-ready chatbot template with authentication, database integration, and deployment configs.",
+    github:
+      "https://github.com/gensx-inc/gensx/tree/main/examples/nextjs-ai-chatbot-template",
+    tags: ["Framework: Next.js", "Category: Template", "Type: Full-stack"],
+  },
+];
+
+export default function ShowcaseClient() {
+  return (
+    <div className="min-h-screen w-full max-w-7xl mx-auto pt-32 px-4 md:px-8 pb-20">
+      <div className="max-w-6xl mx-auto">
+        <AnimatedTitle>Showcase</AnimatedTitle>
+        <motion.p
+          initial={{ opacity: 0, y: 20 }}
+          animate={{ opacity: 1, y: 0 }}
+          transition={{ duration: 0.4, delay: 0.2, ease: "easeOut" }}
+          className="text-lg md:text-xl text-gray-600 text-center mb-12 max-w-3xl mx-auto -mt-12"
+        >
+          Explore demo applications built with GenSX. From chat interfaces to
+          research agents, see what&apos;s possible with our framework.
+        </motion.p>
+
+        <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6">
+          {showcaseItems.map((item, index) => (
```

Comment:
i feel like we should have some visual separation between the full UX examples that people can go play with (the true showcases) and other examples. 

Would also love to have screenshots of the apps (or maybe even gifs) for the web demos as opposed to the gradient backgrounds.

Reply:
Hmm perhaps I can section it off for demos / examples. 

And yeah any of these can have images and it will override the gradient background.